qt/applications/workbench/workbench/plotting/figureinteraction.py

```diff
@@ -704,6 +704,13 @@ def _toggle_normalization(self, ax):
         if ax.lines:  # Relim causes issues with colour plots, which have no lines.
             ax.relim()
 
+        if ax.images:  # Colour bar limits are wrong if workspace is ragged. Set them manually.
+            import numpy as np
```

Comment:
Shouldn't this import be at the top of the file?

Reply:
Damn, that was just for testing and I forgot to move it. You're right.